src/ring/middleware/json.clj

```diff
@@ -6,21 +6,23 @@
             [ring.core.protocols :as ring-protocols]
             [ring.util.io :as ring-io]
             [ring.util.response :refer [content-type]]
-            [ring.util.request :refer [character-encoding]]))
+            [ring.util.request :refer [character-encoding]])
+  (:import
```

Comment:
This `:import` foorm should be on one line.

Reply:
done